test/unit/services/openai-service.test.js

```diff
@@ -0,0 +1,29 @@
+const { generateShortSummary } = require('../../../app/services/openai-service')
+
+jest.mock('@azure/storage-blob')
+jest.mock('../../../app/utils/chunker')
+jest.mock('../../../app/services/blob-client')
+
```

Comment:
Don't forget to mock @langchain/openai - Unit tests should run in isolation even if OpenAI is down

Reply:
This still needs doing

Reply:
oh, I missed this, sorry